protocols/gossipsub/CHANGELOG.md

```diff
@@ -1,15 +1,17 @@
 ## 0.48.0
 
+- Correct state inconsistencies with the mesh and fanout when unsubscribing.
+  See [PR 5690](https://github.com/libp2p/rust-libp2p/pull/5690)
+
 - Deprecate `futures-ticker` and use `futures-timer` instead.
   See [PR 5674](https://github.com/libp2p/rust-libp2p/pull/5674).
+
 - Apply `max_transmit_size` to the inner message instead of the final payload.
   See [PR 5642](https://github.com/libp2p/rust-libp2p/pull/5642).
 
 - Deprecate `void` crate.
   See [PR 5676](https://github.com/libp2p/rust-libp2p/pull/5676).
 
-## 0.47.1
-
```

Comment:
Accidentally removed?

Reply:
I believe its due to the breaking changes from previous PRs that it was decided to skip 0.47.1 to 0.48.0.

Reply:
yeah exactly, thanks Darius!